contentcuration/contentcuration/static/js/edit_channel/import/views/ImportDialogue.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+
+  <div>
+    <!-- SEARCH FORM -->
+    <div>
+      <form @submit="submitSearch" :disabled="!searchTermIsValid" class="SearchForm">
+        <input
+          class="search-input"
+          v-model="searchTerm"
+          type="text"
+          placeholder="What are you looking for?"
+        />
+        <button
+          type="submit"
+          class="action-button modal-main-action-button"
+          @click.prevent="submitSearch"
+          :disabled="!searchTermIsValid"
+        >
+          Search
+        </button>
+      </form>
+    </div>
+
+    <!-- SLOT FOR TREE VIEW OR SEARCH RESULTS -->
+    <div id="import_from_channel_box" class="modal-content-default">
+      <slot />
+    </div>
+
+    <br/>
+
+    <div id="import_bottom_container" class="modal-bottom-content-default">
+      <a class="action-text" data-dismiss="modal">
+        <span>CANCEL</span>
+      </a>
+      <button
+        class="action-button pull-right modal-main-action-button"
+        id="import_content_submit"
+        @click="handleClickImport"
+        :disabled="!importIsEnabled"
+      >
+        <span v-if="!importIsEnabled">
+          Select content to import...
```

Comment:
Would recommend a colon rather than an ellipses, which generally stands in for ommitted content.

Somewhat nerdy and unimportant fact - there's also a single-character ellipses glyph:  

> …

(As far as I know, there's no compelling reason to prefer one over the other.)